cfme/automate/dialogs/dialog_element.py

```diff
@@ -192,10 +193,8 @@ def create(self, element_data):
             view.fill(element)
             view.ele_save_button.click()
         view.save_button.click()
-        try:
-            view.flash.assert_no_error()
-        except AssertionError:
-            raise
+        wait_for(lambda: view.flash.is_displayed, timeout=5, delay=0.2, silent_failure=True)
```

Comment:
Can use `view.flash.wait_displayed(timeout=5)`, if the flash message is_displayed is raising exceptions that don't allow it to run with wait_displayed, we should update the flash message implementation.